spdx-utils/src/main/kotlin/SpdxConstants.kt

```diff
@@ -55,6 +55,16 @@ object SpdxConstants {
      */
     const val REF_PREFIX = "SPDXRef-"
 
+    /**
+     * The prefix to be used for references to other SPDX documents.
+     */
+    const val DOCUMENT_REF_PREFIX = "DocumentRef-"
+
+    /**
+     * The prefix to be used for references to licenses that are not part of SPDX.
```

Comment:
Consider saying "not part of the SPDX license list" ?

Reply:
Done.